--- origsrc/purple-facebook-0.9.0-c9b74a765767/configure.ac	2016-11-21 07:37:22.000000000 -0600
+++ src/purple-facebook-0.9.0-c9b74a765767/configure.ac	2017-03-20 18:11:34.085498300 -0500
@@ -101,4 +101,4 @@ AS_IF(
 
-PLUGIN_CFLAGS="-I`pwd`/$srcdir/include -I`pwd`/$srcdir/pidgin "
-PLUGIN_CFLAGS="${PLUGIN_CFLAGS} -I`pwd`/$srcdir/pidgin/libpurple"
+PLUGIN_CFLAGS="-I\$(top_srcdir)/include -I\$(top_srcdir)/pidgin "
+PLUGIN_CFLAGS="${PLUGIN_CFLAGS} -I\$(top_srcdir)/pidgin/libpurple"
 PLUGIN_CFLAGS="$PLUGIN_CFLAGS -DPURPLE_PLUGINS -include purple-compat.h"
