--- origsrc/purple-facebook-0.9.0-c9b74a765767/configure.ac	2016-11-21 07:37:22.000000000 -0600
+++ src/purple-facebook-0.9.0-c9b74a765767/configure.ac	2017-03-20 18:11:34.085498300 -0500
@@ -80,6 +80,7 @@ AC_ARG_WITH(
 PKG_CHECK_MODULES([GLIB], [glib-2.0 >= 2.28.0 gio-2.0 gobject-2.0])
 PKG_CHECK_MODULES([JSON], [json-glib-1.0 >= 0.14.0])
 PKG_CHECK_MODULES([PURPLE], [purple <  3])
+PKG_CHECK_MODULES([ZLIB], [zlib])
 
 PKG_CHECK_VAR([GLIB_GENMARSHAL], [glib-2.0], [glib_genmarshal])
 AS_IF(
@@ -106,3 +107,3 @@ AS_IF(
 AM_CONDITIONAL([STATIC_FACEBOOK], false)
-AC_SUBST([PLUGIN_LDFLAGS], [-avoid-version])
+AC_SUBST([PLUGIN_LDFLAGS], ["-avoid-version -no-undefined"])
 AC_SUBST([PURPLE_LIBS],    ["$GLIB_LIBS $JSON_LIBS $PURPLE_LIBS $ZLIB_LIBS"])
